ast/desugar/DuplicateHashKeyCheck.cc

```diff
@@ -23,14 +24,18 @@ void DuplicateHashKeyCheck::check(const ExpressionPtr &key) {
     }
 }
 
-// This is only used with Send::ARGS_store and Array::ELEMS_store
 void DuplicateHashKeyCheck::checkSendArgs(const core::MutableContext ctx, int numPosArgs,
                                           absl::Span<const ExpressionPtr> args) {
     DuplicateHashKeyCheck duplicateKeyCheck{ctx};
 
-    // increment by two so that a keyword args splat gets skipped.
-    for (int i = numPosArgs; i < args.size(); i += 2) {
-        duplicateKeyCheck.check(args[i]);
+    args.remove_prefix(numPosArgs);
```

Comment:
Just for my own understanding, this is modifying the span's "view" onto the underlying collection, and doesn't actually change the collection, right?

Reply:
That's correct!